specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscNode.json

```diff
@@ -0,0 +1,957 @@
+              "$ref": "#/definitions/DscNodeUpdateParameters"
+            },
+            "description": "Parameters supplied to the update dsc node."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DscNode"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/nodes": {
+      "get": {
+        "tags": [
+          "DscNode"
+        ],
+        "operationId": "DscNode_ListByAutomationAccount",
+        "description": "Retrieve a list of dsc nodes.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/dscnodeoperations"
+        },
+        "x-ms-examples": {
+          "List DSC nodes by Automation Account": {
+            "$ref": "./examples/listAllDscNodesByAutomationAccount.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DscNodeListResult"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/nodes/{nodeId}/reports": {
+      "get": {
+        "tags": [
+          "NodeReports"
+        ],
+        "operationId": "NodeReports_ListByNode",
```

Comment:
>NodeReports_ListByNode [](start = 24, length = 22)

This one too, x-ms-odata

Reply:
Same as above: Thanks. I'll talk with the team and take this up as a separate PR as it is not a blocker right now.

Reply:
Done

Reply:
Fixed